browser/background/brave_background_mode_manager.cc

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/background/brave_background_mode_manager.h"
+
+#include "brave/browser/profiles/profile_util.h"
+
+BraveBackgroundModeManager::BraveBackgroundModeManager(
+    const base::CommandLine& command_line,
+    ProfileAttributesStorage* profile_storage)
+    : BackgroundModeManager(command_line, profile_storage) {}
+
+BraveBackgroundModeManager::~BraveBackgroundModeManager() {}
+
+// Chromium does not register off-the-record profile with background mode
```

Comment:
Why not? The "dummy" profile might be used in the future for persistent tor profiles, but even if we didn't, why does it matter if it gets registered or not?

Reply:
also this same code will be used for session  profiles that are persistent